src/components/global/header.tsx

```diff
@@ -1,25 +1,40 @@
 import styles from "@/app/page.module.css";
-
+import Image from "next/image";
+import Link from "next/link";
 const Header = () => {
   return (
     <header className={styles.header}>
+      
       {/*This navigation needs to be stuck to the top of the screen when you scroll.*/}
       {/*The header needs to be white and the links need to be in a row.*/}
       <div className={styles.logo}>
         {/* Image needs to be the NIC logo and needs to be linked to the home page of this app. */}
-        <Image />
+        <Link href="/">
+        <Image 
+        src="/NIClogo.png"
+        alt="NIC Logo"
+        height="100"
+        width="300"
+        bg-color="white"/>
+        </Link>
       </div>
       <nav>
-        <ul>
+        <div className={styles['nav-container']}>
+        <ul className={styles['nav-list']}>
           <li>
-            {/*Link to the home page*/}
+            {/* Link to homepage */}
+            <Link href="/">
             Home
+            </Link>
           </li>
           <li>
             {/*Link to the about me page that you have to create.*/}
-            About
+            <Link href= "/app/About.tsx">
```

Comment:
Not how you link.  In this version of nextjs.  About.tsx needed to be in a subfolder called about and needed to be named page.tsx.